lib/handlers/on-search.js

```diff
@@ -279,7 +271,7 @@ module.exports = server => (mailbox, options, session, callback) => {
                                     hdate: !ne
                                         ? entry
                                         : {
-                                              $not: entry
+                                              $not: {}
```

Comment:
This does not seem very logical? 🤔

Reply:
Oh wow. A an accidental typo. I wonder how it even got pushed, I remember fixing it....
Anyway, next commit will fix it